python/lsst/sims/catUtils/utils/testUtils.py

```diff
@@ -1,6 +1,13 @@
+import numpy as np
+import os
+import sqlite3
+import json
 from lsst.sims.photUtils import Bandpass, Sed, PhotometricParameters
+from lsst.sims.utils import (_raDecFromAltAz, _getRotSkyPos, _getRotTelPos, Site,
+                             raDecFromAltAz, haversine, ObservationMetaData)
 
-__all__ = ["calcADUwrapper"]
+
+__all__ = ["calcADUwrapper", "makePhoSimTestDB"]
 
 
 def calcADUwrapper(sedName=None, magNorm=None, redshift=None, internalAv=None, internalRv=None,
```

Comment:
Method probably deserves a doc string, though it was not created on this PR, so maybe later.